app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -637,45 +696,54 @@ $after-button-padding-left: govuk-spacing(5);
 // Dropdown menu items.
 .gem-c-layout-super-navigation-header__dropdown-list-item {
   box-sizing: border-box;
-  padding: 0 0 govuk-spacing(4) 0;
+  padding: 0 0 govuk-spacing(5) 0;
   position: relative;
+  @include govuk-media-query($from: "desktop") {
+    padding: 0 0 govuk-spacing(4) 0;
+  }
 }
 
 // Navigation menu items.
 .gem-c-layout-super-navigation-header__navigation-second-items {
+
   list-style: none;
   margin: 0;
-  padding: govuk-spacing(6) govuk-spacing(4);
+  padding: govuk-spacing(3) govuk-spacing(5) govuk-spacing(5) 0;
 
   & > li {
     margin: 0;
   }
 
   @include govuk-media-query($from: "desktop") {
-    margin: 0 (0 - govuk-spacing(3));
-    padding: govuk-spacing(7) 0 govuk-spacing(8) 0;
+    margin: 0 (0 - govuk-spacing(3)) govuk-spacing(9);
+    padding: govuk-spacing(2) 0 0 0;
 
     & > li {
       margin: 0 govuk-spacing(3);
     }
   }
 }
 
-.gem-c-layout-super-navigation-header__navigation-second-items--topics {
-  @include govuk-media-query($from: "desktop") {
-    @include columns($items: 17, $columns: 2, $selector: "li", $flow: column);
+.gem-c-layout-super-navigation-header__column--government-activity {
+  position: relative;
+
+  @include govuk-media-query($until: "desktop") {
+    &:after {
```

Comment:
Only a small thing but maybe we could leave a comment to explain that the pseudo element is there to only (?) add a border, it took me a little while to work out. Alternatively it would be more consistent with how we do dividers on the [homepage](https://www.gov.uk/) above "Featured" etc. if the divider was added as a top border on the heading, I guess this would require some toggling of a special class on `.gem-c-layout-super-navigation-header__column-header`. However a class might be more explicit in its function than a pseudo element (it would be neat if the [heading with border component](https://components.publishing.service.gov.uk/component-guide/heading/with_border/preview) admitted specific border styles, I've just raised this in a separate channel).

Reply:
ah yep, i can actually totally remove that and use a border instead (not sure why i didn't do this before, i think there was a redesign where this line was going to change to a vertical one on desktop)